spec/swagger.yaml

```diff
@@ -15,57 +15,40 @@ tags:
   - name: products
   - name: users
   - name: audits
+  - name: server
+  - name: image
+
 paths:
-  /info.json:
+  /info:
     get:
-      description: global server Information
+      tags:
+        - server
+      description: global server information and capabilities
       responses:
         200:
-          description: information
+          description: global server information
           schema:
-            type: object
-            required:
-              - version
-            properties:
-              version:
-                type: string
-                example: "2.1"
-  /audits.json:
+            type: array
+            items:
+              $ref: "#/definitions/Server"
+  /audits:
     get:
       tags:
         - audits
       summary: statistics about previous transactions
-      description: |
-        statistics about previous transactions
+      description: statistics about previous transactions
       parameters:
       - in: query
-        name: start_date[year]
-        required: false
-        type: integer
-      - in: query
-        name: start_date[month]
-        type: integer
-        minimum: 1
-        maximum: 12
-      - in: query
-        name: start_date[day]
-        type: integer
-        minimum: 1
-        maximum: 31
+        name: start
+        required: true
+        type: string
+        description: start date in ISO8601 format
```

Comment:
I really like this. 😃 
The date format is currently very ugly.

Reply:
Yes it's ugly and these parameter arrays are only supported by php and ruby. Afaik no other language is capable of handling these properly.

Reply:
We could use `type: "date"` for this. (see [Swagger Spec](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#data-types) and [RFC 3339](https://xml2rfc.tools.ietf.org/public/rfc/html/rfc3339.html#anchor14))

Reply:
yes, I'm still learning swagger and when I raised the pull request, I didn't know about it :(